.github/workflows/build.yaml

```diff
@@ -1,5 +1,5 @@
 ################################################################################
-# Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+# Copyright (c) 2021,2024 Contributors to the Eclipse Foundation
```

Comment:
Hi, you've misunderstood me regarding the copyright header.

Only the files which have been updated during the commit, get the updated year. Please revert any files that did not change.

Reply:
Most of the files are modified in this PR,  if not #77 will do it.

Reply:
If the only thing that changed in a file is the year in the copyright header, it shouldn't be part of the PR. Regardless of other PRs.